components/FileUpload.vue

```diff
@@ -56,6 +56,12 @@ const onUpload = async (event: Event) => {
 </script>
 <template>
   <div class="flex-center w-full">
+    <div class="flex justify-center mb-8 text-center">
+      <h1 class="text-3xl max-w-[418px] font-medium text-pretty tracking-wide">
+        Generate <span class="text-[#c875f4]">beautiful</span> color palette from a image
+      </h1>
+    </div>
+    
```

Comment:
You can remove the wrapping `div` element and put `mb-8` and `text-center` classes directly in `h1`:
```suggestion
    <h1 class="mb-8 text-center mx-auto text-3xl max-w-[418px] font-medium text-pretty tracking-wide">
        Generate <span class="text-[#c875f4]">beautiful</span> color palettes from an image
    </h1>
```